<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>992</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>60</y>
         <width>1281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>WBCs - LCCs DETECT SYSTEM</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QFrame" name="img_detect">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>640</width>
         <height>640</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="img_upload">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>640</width>
          <height>640</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>160</y>
         <width>521</width>
         <height>640</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <widget class="QWidget" name="browse_img_frame" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>521</width>
          <height>41</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>91</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Choose Image:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="browseBtn">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>10</y>
           <width>113</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="pathLabel">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>300</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="frameChooseModel" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>501</width>
          <height>41</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Choose Model:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioBtnFaster">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Faster RCNN</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioBtnYolov8">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>10</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>YOLOv8</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioBtnDeTr">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>10</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>DeTr</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="statisticalLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>340</y>
          <width>521</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>STATISTICAL TABLE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="img_process_label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>521</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>No image available for processing!!!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>380</y>
          <width>221</width>
          <height>201</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTableWidget" name="statisticalTable">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>221</width>
           <height>201</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QLabel" name="statusProcessLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>290</y>
          <width>521</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="chooseTypeWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>99</y>
          <width>501</width>
          <height>41</height>
         </rect>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Choose Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="chooseTypeCombo">
         <property name="geometry">
          <rect>
           <x>99</x>
           <y>0</y>
           <width>311</width>
           <height>41</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>White Blood Cells</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lung Cancer Cells</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1287</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
